.github/comment.yml

```diff
@@ -77,6 +77,14 @@
     onLabeledComments: "Please ensure to respond feedbacks from the ARM API reviewer. When you are ready to continue the ARM API review, please remove `ARMChangesRequested`"
     onLabeledRemoveLabels:
         - WaitForARMFeedback
+
+- rule:
+    type: label
+    label: Approved-BreakingChange
+    booleanFilterExpression: !(ARMSignedOff||ARMChangesRequested||Approved-OkToMerge||WaitForARMRevisit)
```

Comment:
How about the case if PR author fixed breaking change by revert some change.  it means BreakingChangeReviewRequired removed.  WaitForARMFeedback will be automatic added back?

or in this case, if ARM is not need to review. it will add ARMReview label?

Reply:
If BreakingChangeReviewRequried removed, WaitForARMFeedback will not be added. 

Updated. This rule is to auto add label "WaitForARMFeedback" when breakingChange approved and PR with "ARMReview" label.  ARMReview label will be added by pipeline PR summary check which indicated the PR actually need arm review.